examples/cli_tutorial.ipynb

```diff
@@ -182,8 +182,8 @@
    "outputs": [],
    "source": [
     "# set calculation and database path for our project \"wgs\"\n",
-    "!mkdir wgs_project/calc\n",
-    "!mkdir wgs_project/db\n",
+    "!mkdir -p wgs_project/calc\n",
```

Comment:
Do we need the -p option for mkdir?

Reply:
Yup, since we didn't create the `wgs_project` folder yet, so -p creates both wgs_project and wgs_project/calc folders at the same time